docs/book/aya/crosscompile.md

```diff
@@ -0,0 +1,27 @@
+# Cross-compiling aya-based programs
+
+The instructions below show how to cross compile aya eBPF programs on Macs.
+Cross compiling on other systems is possible too and we're going
+to add instructions to do that soon (PRs welcome!).
+
+# Cross-compiling aya-based programs on Mac
+
+Cross compilation should work on both Intel and Apple Silicon Macs.
+
+1. Install `rustup` following the instructions on <https://rustup.rs/>
+2. Install a rust stable toolchain: `rustup install stable`
+3. Install a rust nightly toolchain:  
+`rustup toolchain install nightly --component rust-src`
+4. `brew install llvm`
```

Comment:
Install LLVM from brew: ...
Install the musl cross compiler: ...

Otherwise it's weird that in some points you explain what a command does, and in some you don't

Reply:
done - also added the missing `rustup target` instructions